lib/fizzy/types.hpp

```diff
@@ -212,6 +215,14 @@ struct Code
     bytes immediates;
 };
 
+// https://webassembly.github.io/spec/core/binary/modules.html#data-section
+struct Data
+{
+    MemIdx memidx;
```

Comment:
```suggestion
    MemIdx memory_index;
```
(we have `function_type_index` in another struct)